zerver/tests/test_events.py

```diff
@@ -2402,10 +2402,10 @@ def test_rename_stream(self) -> None:
             ('stream_id', check_int),
         ])
         schema_checker('events[1]', events[1])
-        schema_checker = check_dict([
+        schema_checker = check_dict_only([
```

Comment:
Outer schema checkers should be `check_events_dict`, which automatically adds the `id` field and is also a hook for further validation (i.e. likely where we'll check OpenAPI records).  I'll just fix this before merging.

FYI @orientor.